plugins/lookup/conjur_variable.py

```diff
@@ -66,6 +56,78 @@
             key: config_file_path
         env:
           - name: CONJUR_CONFIG_FILE
+      conjur_appliance_url:
+        description: Conjur appliance url
+        type: string
+        default: 
```

Comment:
I think we can remove these empty `default: ` fields

Reply:
We will remove the empty default: fields